### PR TITLE
#10 Feat: Add Projects Section

### PR DESCRIPTION
**Pull request by mercihabam**
### Task Description _(❗️Required)_

Added a projects section with links to the individual projects. On hover, a project overview displays on the right side on larger screen whereas on smaller screen, the project overview is displayed under the project link after the user clicks on the link
### Type of Task

- [ ] fix
- [ ] chore
- [x] feat
- [ ] refactor
- [ ] perf
- [ ] docs

### Guidelines

- [x] Does your code follow ES6+ syntax?
- [x] Did you use arrow functional components?
- [x] Did you thoroughly review your code?
- [x] Have you checked if your code does not introduce conflicts in the main codebase?

### Ticket Link _(❗️Required)_

https://www.notion.so/apeunit/Projects-Section-2bf98fd74be2401cbd23030091158b5e

### CODE REVIEW

File: components/sections/Projects/ProjectTitle.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <div className="text-13 leading-16 sm:text-16 sm:leading-18">{name}</div>
+      <div className="w-full absolute bottom-1 left-1.5">
+        <div className="opacity-0 w-full border-b-2 border-black group-hover:opacity-100 transition-all hidden xl:block"></div>
+      </div>
+      <Image
+        src="/assets/icons/arrow-link.svg"
+        alt="arrow"
+        width={14}
+        height={14}
+        className=" opacity-0 group-hover:opacity-100 transition-opacity duration-300 hidden xl:block mt-[14px]"
```
Comment (Lily-Lilian) [inline components/sections/Projects/ProjectTitle.jsx]: Try to use rem and define this specifically by extending the Tailwind CSS configuration and remove this arbitrary values.

**Review by Odile078 — COMMENTED** [review]
You did a great job, only one thing to consider. Consider creating a data folder outside of the components folder and put in there the ProjectsData file. This is because The helpers folder shouldn't be in the components folder since it is not a component. and also the data folder shouldn't be in helpers since the helpers folder contains files that have functionalities to be used across the project while the data folder contains the data that is going to be needed in the project.

**Review by Lily-Lilian — DISMISSED** [review]
LGTM!

**Review by Odile078 — DISMISSED** [review]

**Review by Odile078 — DISMISSED** [review]

**Review by Lily-Lilian — DISMISSED** [review]
Well done!!

File: components/sections/Projects/ProjectTitle.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+const ProjectTitle = ({ name, openProjectPage, width, active, index }) => {
+
+  return (
+    <div
+      className="cursor-pointer relative overflow-hidden  group flex items-start"
+      onClick={width > 1024 ? openProjectPage: () =>{}}
+    >
+      <div className="text-13 leading-16 sm:text-16 sm:leading-18">{name}</div>
+      <div className={'transition-opacity duration-300 ' + (active === index ? 'opacity-100': 'opacity-0')}>
```
Comment (alain-kubwayo) [inline components/sections/Projects/ProjectTitle.jsx]: I think if you are using double quotes, it's good to stay consistent throughout. You should change single quotes into double quotes to keep consistency.

**Review by alain-kubwayo — CHANGES_REQUESTED** [review]
Good job!
```
I left a comment

**Review by alain-kubwayo — APPROVED** [review]
Looks good! 🚀

**Review by Lily-Lilian — APPROVED** [review]

**Review by Odile078 — APPROVED** [review]

**Pull request merged**